models/grouped_assignments.py

```diff
@@ -7,7 +7,7 @@
     Field("course", db.courses),
     Field("name", "string"),
     Field(
-        "points", "integer", default=0, notnull=True
```

Comment:
This needs to be put back.  I can walk you through updating your database so it won't cause you problems.